views/pages/home.html

```diff
@@ -2,7 +2,8 @@
 <div class="jumbotron">
 	<h1><span class="accented">Unicorn</span> Rampage</h1>
 	<p class="header-caption">The modern day dungeon master helpmate</p>
-	<button class="btn btn-default" type="button">Create</button>
-	<button class="btn btn-default" type="button">Play</button>
+	<a ng-href="/#/create"><button class="btn btn-default" type="button">Create</button></a>
+	<a ng-href="/#/play">
```

Comment:
Hey thanks for the feedback.

I used /#/ as the prefix only because that's what they used in the Code School tutorials. I have a router set up in /scripts/app.js that should take any /play and import the play.html template. However, I just tested it in localhost and it does not import properly unless I specify the /#/.